### DISCUSSION TITLE
Flunt

### DISCUSSION THREAD

**Discussion by ClaudineiMonteiro** [balta-io/forum#120] [Tira Dúvidas] [answered]
Fui tentar reproduzir o conteudo do curso, porém a parte de validação com o Flunt, apartir da verão 1.5 mudou, como devo fazer a validação agora?
Estou fazendo o projeto com .net7.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @ClaudineiMonteiro, como vai?

Você pode adicionar uma versão específica de um pacote utilizando a flag `--version`. No caso do curso, ele usa a versão 1.0.5 e você pode instalar ela assim 👇

```
dotnet add package Flunt --version 1.0.5
```

Se quiser utilizar a versão 2.X, aqui tem um guia:
https://github.com/andrebaltieri/Flunt

PS:. Recomendo usar a mesma versão do curso... depois você tenta migrar...

**Reply by ClaudineiMonteiro** [reply]
Bom dia,
Muito obrigado pela resposta, me ajudou muito, depois vou tentar usar a ultima versão.
